android/lib/build.gradle

```diff
@@ -15,7 +15,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:+'
+  compile 'com.facebook.react:react-native:0.29.0'
   compile "com.google.android.gms:play-services-base:8.4.0"
```

Comment:
You can't know for sure...  I believe you are more or less asking for something similar to "Peer Dependencies" in NPM. I don't believe there is anything like that for gradle.

According to @felipecsl, If we specify `react-native:0.29.0` here, if another project uses `react-native:0.30` or something, gradle will only choose to use `0.30`. Perhaps Felipe can confirm.